src/Servers/Kestrel/tools/CodeGenerator/HttpProtocolFeatureCollection.cs

```diff
@@ -26,6 +26,7 @@ public static string GenerateFile()
             var commonFeatures = new[]
             {
                 "IHttpAuthenticationFeature",
+                "IHttpRequestTrailersFeature",
```

Comment:
I think all the `implementedFeatures` should also be `alwaysFeatures` instead of `commonFeatures`.